src/Sulu/Bundle/ContactBundle/Contact/ContactManager.php

```diff
@@ -81,31 +66,22 @@ class ContactManager extends AbstractContactManager implements DataProviderRepos
      */
     protected $userRepository;
```

Comment:
Sadly we can't promote protected properties as this would enforce the types on the properties (and be a bc break). I don't think they should have been protected in the first place though.

Reply:
We could still promote them just with phpdoc instead of hardcoded type.

Reply:
This would remove the typehint for the constructor but once all types and the doc comments are gone, this should be a good solution.

Reply:
Alternatively, I don't think this is a huge bc break. Because if you put any other classes into that property the class is going to crash anyways. So I think promoting that and keeping the typehint should be fine.

Reply:
Will do, it's probably not the last time that I'll go over classes. And specific types should be replaced with interfaces whereever possible anyways.